TheInternet/HoversPage.cs

```diff
@@ -0,0 +1,51 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+
+namespace TheInternet;
+
+public class HoversPage : BasePage
+{
+    IList<IWebElement> userImagesList => driver.FindElements(By.CssSelector("div [class = 'figure']"));
+   
+    public HoversPage(IWebDriver driver) : base(driver)
+    {
+        this.driver = driver;
+    }
+
+    public UserCaption hoverOveruserImage(int userNumber)
+    {
+        userNumber = userNumber - 1;
+        Actions actions = new Actions(driver);
```

Comment:
Convention/preference option:

Many teams use `var` instead of explicitly specifying the type (the first `Actions` in this case).

This is a simple case were the type you're `new`ing up is clear on the right, but the example below would require you to go find out what `ElementAt` returns, var saves you that bother, but it's 100% a preference thing and it's fine to mix the two.